app/services/magic_link_sign_up.rb

```diff
@@ -1,7 +1,15 @@
 class MagicLinkSignUp
   def self.call(candidate:)
+    @candidate = candidate
     magic_link_token = MagicLinkToken.new
-    AuthenticationMailer.sign_up_email(to: candidate.email_address, token: magic_link_token.raw).deliver_now
-    candidate.update!(magic_link_token: magic_link_token.encrypted, magic_link_token_sent_at: Time.now)
+    AuthenticationMailer.sign_up_email(to: @candidate.email_address, token: magic_link_token.raw).deliver_now
+    @candidate.update!(magic_link_token: magic_link_token.encrypted, magic_link_token_sent_at: Time.now)
+    notify_slack
+  end
+
+  def self.notify_slack
+    text = "New sign-up [candidate_id: #{@candidate.id}]"
+    url = Rails.application.routes.url_helpers.support_interface_applications_url
+    SlackNotificationWorker.perform_async(text, url)
```

Comment:
This limits us to a single line that is linked. Should we move the URL generation into these methods so we can do `SlackNotificationWorker.perform_async(some_markdown)`?

Reply:
discussed IRL, refactoring